src/components/Home/Home.js

```diff
@@ -1,26 +1,30 @@
-import React from "react";
+import React, { useEffect } from "react";
 import Main from "../Main/Main";
 import Footer from "../Footer/Footer";
-// import api from "../utils/api";
-// import { useEffect } from "react";
-// import { useHistory } from "react-router-dom";
 import About from "../About/About";
 import SigninPopup from "../SigninPopup/SigninPopup"
 import SignupPopup from "../SignupPopup/SignupPopup";
 import SearchResults from "../SearchResults/SearchResults";
 import Preloader from "../Preloader/Preloader";
 import ResultNotFound from "../ResultNotFound/ResultNotFound";
 import SignupSuccess from "../SignupSuccess/SignupSuccess";
+import newsapi from "../../utils/NewsApi";
+import { CurrentUserContext } from "../../contexts/CurrentUserContext";
+import mainapi from "../../utils/MainApi";
 
 function Home(props) {
 
     const [isSigninPopupOpen, setIsSigninPopupOpen] = React.useState(false);
     const [isSignupPopupOpen, setIsSignupPopupOpen] = React.useState(false);
     const [isSignupSuccessOpen, setIsSignupSuccessOpen] = React.useState(false);
+    const [articles, setArticles] = React.useState([]);
+    const [isNotFound, setIsNotFound] = React.useState(false);
+    const [currentUser, setCurrentUser] = React.useState({});
 
-    const headerPopup = document.querySelector(".popup-header");
+    
 
     function handleSigninClick() {
+      const headerPopup = document.querySelector(".popup-header");
```

Comment:
`document.querySelector` shouldn't be used in `React` apps because `React` was specially created no to search anything in the DOM. 

React knows every element because it renders them in its components. Please, correct it in your whole project.